src/fragments/lib-v1/ssr/nextjs/getting-started.mdx

```diff
@@ -0,0 +1,335 @@
+
+</Callout>
+
+## Using Amplify Category APIs as a Data Source on The Server Side
+
+In general, to use Amplify APIs on the server in your Next.js app, you will
+need to:
+
+1. Import the API from the `/server` sub path.
+2. Using the `runWithAmplifyServerContext` helper function exported from
+`@aws-amplify/adapter-nextjs` to call the Amplify API in an isolated server context.
+
+<Callout>
+
+**NOTE:** A subset of Amplify APIs can now be called on the server side of a Next.js
+app. These APIs are exported from the `/server` sub paths. See
+[the full list](/lib-v1/ssr/nextjs/q/platform/js/#amplify-api-nextjs-server-support)
+of supported APIs.
+
+</Callout>
+
+### With Next.js App Router
+
+#### In React Server Component
+
+##### Dynamic Rendering
+
+Dynamic rendering is based on a user session extracted from an incoming request.
+
+```typescript
+import { cookies } from 'next/headers';
+import { runWithAmplifyServerContext } from '@aws-amplify/adapter-nextjs';
+import { getCurrentUser } from '@aws-amplify/auth/server';
+
+// This page always dynamically renders per request
+export const dynamic = 'force-dynamic';
+
+export default async function AuthGetCurrentUserServer() {
+  try {
+    const currentUser = await runWithAmplifyServerContext({
+      nextServerContext: { cookies },
+      operation: (contextSpec) => getCurrentUser(contextSpec),
+    });
+
+    return (
+      <AuthFetchResult
+        description="The API is called on the server side."
+        data={currentUser}
+      />
+    );
+  } catch (error) {
+    console.error(error);
+    return <p>Something went wrong...</p>;
+  }
+}
+```
+
+##### Static Rendering
+
+Static rendering doesn’t require a user session, so you can specify the
+`nextServerContext` parameter as `null`. This is useful for some use cases,
+for example, when you are using the Storage API with guest access (if you have
+enabled it in your backend).
+
+```typescript
+import { runWithAmplifyServerContext } from '@aws-amplify/adapter-nextjs';
+import { getUrl } from 'aws-amplify/storage/server';
+import Image from 'next/image';
+
+// Re-render this page every 60 minutes
+export const revalidate = 60 * 60; // in seconds
+
+export default async function StaticallyRenderedPage() {
+  try {
+    const splashUrl = await runWithAmplifyServerContext({
+        nextServerContext: null,
+        operation: (contextSpec) =>
+        getUrl(contextSpec, {
+            key: 'splash.png',
+        }),
+    });
+    
+    return (
+      <Image
+        src={splashUrl.url.toString()}
+        alt="Splash Image"
+        width={500}
+        height={500}
+      />
+    );
+  } catch (error) {
+    console.error(error);
+    return <p>Something went wrong...</p>;
+  }
+}
+```
+
+<Callout>
+
+**NOTE:** The URL returned by the `getUrl` API expires in the above example.
+You may want to specify the `revalidate` parameter to rerender the page as
+required to ensure the URL gets regenerated.
```

Comment:
Nice that the `revalidate` works! Does it work both Vercel and Amplify hosting?

Reply:
Yes I think so, as it's environment independent